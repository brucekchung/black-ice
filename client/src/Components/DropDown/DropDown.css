.DropDown {
  display: inline;
  margin: 0 10px;
}