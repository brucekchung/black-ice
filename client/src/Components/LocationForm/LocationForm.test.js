/* eslint-disable */
import React from 'react'
import DropDown from './LocationForm'
import { shallow } from 'enzyme'

describe('DropDown', () => {
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(<DropDown options={['Argentina', 'Peru']}
                                name='Countries'
                                handleChange={ jest.fn() }/>)
  })

  it('should match snapshot upon load', () => {
    expect(wrapper).toMatchSnapshot()
  })

  it('should render the options that are passed down', () => {
    const options = ['Argentina']
    const rendered = wrapper.instance().renderOptions()
    const expected = [
      <option value='Argentina' key='Argentina0'>Argentina</option>,
      <option value='Peru' key='Peru1'>Peru</option>
    ]

    expect(rendered).toEqual(expected)
  })

})
