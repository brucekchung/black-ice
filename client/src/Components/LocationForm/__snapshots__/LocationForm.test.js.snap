// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DropDown should match snapshot upon load 1`] = `undefined`;
